src/modules/hunting/dns.py

```diff
@@ -0,0 +1,66 @@
+import re
+import logging
+
+from ...core.events import handler
+from ...core.events.types import Event, Vulnerability
+from ...core.types import ActiveHunter, KubernetesCluster, IdentityTheft
+
+from .arp import PossibleArpSpoofing
+
+from scapy.all import IP, ICMP, UDP, DNS, DNSQR, ARP, Ether, sr1, srp1, srp
+
+class PossibleDnsSpoofing(Vulnerability, Event):
+    """A malicous pod running on the cluster could potentially run a DNS Spoof attack and perform a MITM attack on applications running in the cluster."""
+    def __init__(self, kubedns_pod_ip):
+        Vulnerability.__init__(self, KubernetesCluster, "Possible DNS Spoof", category=IdentityTheft)
+        self.kubedns_pod_ip = kubedns_pod_ip
+        self.evidence = "kube-dns at: {}".format(self.kubedns_pod_ip)
+
+# Only triggered with RunningAsPod base event
+@handler.subscribe(PossibleArpSpoofing)
+class DnsSpoofHunter(ActiveHunter):
+    """DNS Spoof Hunter
+    Checks for the possibility for a malicous pod to compromise DNS requests of the cluster (results are based on the running node)
+    """
+    def __init__(self, event):
+        self.event = event
+    
+    def get_cbr0_ip_mac(self):
+        res = srp1(Ether() / IP(dst="1.1.1.1" , ttl=1) / ICMP(), verbose=0)
+        return res[IP].src, res.src
+
+    def extract_nameserver_ip(self):
+        with open('/etc/resolv.conf', 'r') as f:
+            # finds first nameserver in /etc/resolv.conf
+            match = re.search(r"nameserver (\d+.\d+.\d+.\d+)", f.read())
```

Comment:
`nameserver` may be followed by tabs or more spaces.
Make the pattern to catch any white space after it.